src/mono/mono/metadata/marshal.c

```diff
@@ -6580,3 +6580,271 @@ mono_wrapper_caches_free (MonoWrapperCaches *cache)
 	free_hash (cache->thunk_invoke_cache);
 	free_hash (cache->unsafe_accessor_cache);
 }
+
+#ifdef MONO_ARCH_HAVE_SWIFTCALL
+typedef enum {
+	SWIFT_EMPTY,
```

Comment:
What is the difference between SWIFT_OPAQUE and SWIFT_EMPTY?

Reply:
Opaque: This byte is used by a field that and doesn't meet any other classification

Empty: This byte is only used for padding, there's no field here.